coderd/workspacequota.go

```diff
@@ -0,0 +1,37 @@
+package coderd
+
+import (
+	"net/http"
+
+	"github.com/coder/coder/coderd/database"
+	"github.com/coder/coder/coderd/httpapi"
+	"github.com/coder/coder/coderd/httpmw"
+	"github.com/coder/coder/coderd/rbac"
+	"github.com/coder/coder/codersdk"
+)
+
+func (api *API) workspaceQuota(rw http.ResponseWriter, r *http.Request) {
```

Comment:
I don't believe we need this endpoint in AGPL... we already fetch entitlements, which should determine whether the feature is enabled or not. If not, we just don't need to request (unless this isn't a noop and I'm misunderstanding).

![image](https://user-images.githubusercontent.com/7122116/193127984-4b17756f-5a19-44b1-b6cb-76f6468a86b4.png)

Reply:
Yeah so this endpoint just returns a limit of 0 for AGPL. We don't need the endpoint in AGPL if we just catch the 404 and handle that on the client, but I wasn't sure if that was cleaner or not. I'm down to move to ent only.

Reply:
Well I guess I'm saying we never need to hit the endpoint, because we already fetch entitlements before-hand.